pkg/kubelet/volumes.go

```diff
@@ -34,8 +34,6 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 )
 
-var errUnsupportedVolumeType = fmt.Errorf("unsupported volume type")
-
```

Comment:
Hmm.. do we really not need this or can we move it somewhere where we are sure that the volume is not supported ?